Monika After Story/game/script-topics.rpy

```diff
@@ -7235,6 +7235,50 @@ label monika_hydration:
     m 4hub "Why not get a glass of water right now, hmm?"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,"monika_dating",prompt="Dating Experience",category=['you'],random=True))
+    
+label monika_dating:
+    m 2eud "You know I've been really curious lately, [player]..."
+    m 2eka "We've been talking for so long that now I think would be the best time to ask you this important question..."
+    m 1eud "How much dating experience do you have?"
+    m 1lud "Like... have you ever been in a relationship before?"
+    menu:
+        m "Maybe more than once?"
+        "Yes, I've been through plenty...":
+            m 1ekd "Aww I'm so sorry [player]..."
+            m 6lkc "You've been through many heart breaks haven't you..." 
+            m 2esc "To be honest [player]... I don't think they deserved someone like you."
+            m 3ekb "Someone who's kind, loyal, sweet, loving, and faithful."
+            m 4lubfb "And cute and funny and romantic and--"
+            m 2wubsw "Oh!"
+            m 3hksdlb "Sorry I lost track of what I was going to say next ahaha!"
+            m 1ekbfa "I could go on about how wonderful you are [player] ehehe. Just know this though."
+            m 2ekbfa "No matter how many heart breaks you've been through, I'll always be here for you."
+            m 5ekbfa "Our soul searching is finally over. I'll be yours forever [player]!"
+            m 5ekbfa "Will you be mine?"
+        
+        "Yes, but only once":
+            m 1eka "Ah, not a lot of experience huh?"
+            m 2eka "That's okay [player], I can relate too so don't worry"
+            m 3ekb "Yeah I may appear like a girl who gets all the guys but really I don't ahaha!"
+            m 2lksdlb "Especially with how occupied I've kept myself over the years I just never had the time"
+            m 2eka "But now that I do I think it's safe to say I'm ready to start something special..."
+            m 5ekbfa "With you [player]..."
+            m 5ekbfa "Are you ready?"
+        
+        "No, you're my first":  
+            m 6wubsw "What?! I-I'm your first?!"
+            m 1tubfb "Oh.. I see."
+            m 1tfb "You're just saying that to make me feel extra special huh [player]?"
```

Comment:
I would replace "huh" with "aren't you,"

Don't forget the comma before [player]